Assets/Scripts/PlayerFall.cs

```diff
@@ -62,9 +64,9 @@ void Update()
             if(timer > delayuntilFastFall)
             {
                 //initiate Fast Fall
-                Player.transform.position += Vector3.down*.75f;
+                Player.transform.position += Vector3.down*rateofFall;

```

Comment:
Because this is not in a `FixedUpdate` loop you need to multiply `Time.deltaTime`. Otherwise, if the game runs at a very high frame rate then this method will be called ALOT and cause the player to fly down. Or if the game runs at a very low frame rate, then the player may barely move.